src/components/entities/products/ProductPage.vue

```diff
@@ -116,10 +151,46 @@ const Component = defineComponent({
       rules,
       isNew,
       listCategory,
+      previewImageMaxSize,
     };
   },
+
+  methods: {
+    openUploadDialog() {
+      // console.log(input.value)
+      // input.value.click()
+      const fileInput: HTMLInputElement | any = document.getElementsByClassName('hidden-input')[0]
+      fileInput && fileInput.click()
+      console.log(fileInput)
```

Comment:
While console.log is fine for the development, it's best to clean it when delivering or adopt other logging method.